rhine-bayes/src/FRP/Rhine/Bayes.hs

```diff
@@ -11,10 +11,10 @@ import Control.Monad.Bayes.Class
 import Control.Monad.Bayes.Population
 
 -- dunai
-import qualified Control.Monad.Trans.MSF.Reader as DunaiReader
+import qualified Data.Automaton.MSF.Trans.Reader as MSFReader
 
 -- dunai-bayes
-import qualified Data.MonadicStreamFunction.Bayes as DunaiBayes
+import qualified Data.MonadicStreamFunction.Bayes as MSFBayes
 
```

Comment:
Fix import comments

Reply:
Have you considered using [`PackageImports`](https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/package_qualified_imports.html)?

Reply:
Yes, I've even tried this in another project. I wasn't sure whether it's worth it, especially because the extension is not so well-known, and I'd have to add the package name to every import.